notes/programming-languages/python/packages/pandas.md

```diff
@@ -0,0 +1,123 @@
+# Python Language Overview
+
+### The `pandas` Module
+
+The `pandas` module is an extremely useful one for working with structured data. You can think of `pandas` as a python tool for creating and manipulating powerful spreadsheets-like objects called DataFrames. `pandas` also includes some SQL-like features and can be used to easily read and write data stored in csv's or databases.
+
+Reference:
+
+  + http://pandas.pydata.org/pandas-docs/stable/
+
+> NOTE: `pandas` supports Python 2.7 & 3.4+
+
+First install the package using pip, if necessary:
+
+```` sh
+# install requests
```

Comment:
consider omitting this commented-out section line to avoid confusion. or change the package name, but I think the preceding markdown is sufficient to say what we're doing here.